vue-app/src/components/UI/FeatureDescription.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+    <div class="container">
+        <h1>{{ site.label }}</h1>
+        <div class="site-description">
+            <button :class="{inactive: isFirst}" @click="onPreviousClick">&lt;</button>
+            <p>
+                {{ site.description }}
+            </p>
+             <button :class="{inactive: isLast}" @click="onNextClick">&gt;</button>
+        </div>
+        <router-link :to="site.link"> Go </router-link>
+    </div>
+</template>
+
+<script setup lang="ts">
+import { FeatureDescriptionProps } from '../../models/FeatureDescriptionProps';
+
+const { site, isFirst, isLast } = defineProps<FeatureDescriptionProps>();
+const emit = defineEmits(['next', 'previous']);
+function onPreviousClick() {
+    if(isFirst) return;
```

Comment:
something is wrong 'cause I can't go back from the second site desc to the first one, isFirst is true